lib/Serialization/SerializeDoc.cpp

```diff
@@ -394,7 +357,9 @@ static void writeDeclCommentTable(
       if (!D->canHaveComment())
         return false;
 
-      // Skip the decl if it does not have a comment.
+      // Skip the decl if it does not have a comment. Note this means
+      // we'll only serialize "direct" brief comments, but that's okay
+      // because clients can compute the semantic brief comment themselves.
```

Comment:
Seems weird that we output the brief comment to me at all. @nkcsgexi do you remember why this is the case?